test/test_other.py

```diff
@@ -8620,25 +8620,18 @@ def test_legalize_js_ffi(self):
         assert not e_f32_f64, 'f32 converted to f64 in exports'
         assert e_i64_i64,     'i64 converted to i64 in exports'
 
+  @disabled('https://github.com/WebAssembly/binaryen/pull/6428')
```

Comment:
Maybe just delete it, as we'll not want to re-enable it?

Reply:
I guess its still tests the `-sLEGALIZE_JS_FFI=0` setting itself.   I guess we should remove this once we remove that setting?